apps/api/src/app/user/update-user-setting.dto.ts

```diff
@@ -52,6 +52,10 @@ export class UpdateUserSettingDto {
   @IsOptional()
   savingsRate?: number;
 
+  @IsNumber()
+  @IsOptional()
+  targetNetWorth?: number;
```

Comment:
I would rename to `projectedTotalAmount` (or do you know a better name? e.g. `retirementAmount`).

Reason: In my case it is not the net worth as I have an emergency fund (it is not included here, because the annual interest rate of the emergency fund is much lower).

Reply:
Done, used projectedTotalAmount in latest commit